src/styles/blocks/review.css

```diff
@@ -0,0 +1,34 @@
+.review {
+  align-self: flex-start;
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
+  width: 100%;
+  height: 16px;
+}
+
+.stars {
+  display: flex;
+  flex-direction: row;
+}
```

Comment:
Star is a separate block. Maybe it deserves a separate CSS file.

Reply:
Right ....Done!